### PR TITLE
Solid Colored Axons, and a couple of other things.

### PR DESCRIPTION
**Pull request by UnknownEvil**
Changes

### CODE REVIEW

File: src/evolv/io/Brain.java [hunk 1]
```diff
@@ -5,19 +5,19 @@
 import evolv.io.util.MathUtil;
 
 public class Brain {
-	private static final int BRAIN_HEIGHT = Configuration.NUM_EYES * 3 + Configuration.MEMORY_COUNT + 6;
+	private static int BRAIN_HEIGHT = Configuration.NUM_EYES * 3 + Configuration.MEMORY_COUNT + 3 < 9 ? 9 : Configuration.NUM_EYES * 3 + Configuration.MEMORY_COUNT + 3;
```
Comment (NateUniAccount) [inline src/evolv/io/Brain.java]: The
3 < 9 ? 9 : Configuration.NUM_EYES * 3 + Configuration.MEMORY_COUNT + 3;
part can all be replaced with
9;

**Comment by NateUniAccount** [comment]
Looks good, I prefer the line colors and the outlines but that's just my personal preference

File: src/evolv/io/Brain.java (hunk 1 above)
Comment (kodomazer) [inline src/evolv/io/Brain.java]: It's possible that he's actually missing a set of parentheses around the entire preceding portion of the statement so 
` Configuration.NUM_EYES * 3 + Configuration.MEMORY_COUNT + 3 < 9 ? 9 : Configuration.NUM_EYES * 3 + Configuration.MEMORY_COUNT + 3;`
was meant to be
 `(Configuration.NUM_EYES * 3 + Configuration.MEMORY_COUNT + 3 )< 9 ? 9 : (Configuration.NUM_EYES * 3 + Configuration.MEMORY_COUNT + 3);`
In order to keep the minimum height at 9

Comment (UnknownEvil) [inline src/evolv/io/Brain.java]: There is no need for the parenthesis, but it should not all be replaced with 9, that line is to make sure it that number is dynamic, but goes no lower than 9.

Comment (NateUniAccount) [inline src/evolv/io/Brain.java]: It's possible to do it so it's easier to work out what is going on, it just needs a method which BRAIN_HEIGHT calls, something like BRAIN_HEIGHT = calculateBrainHeight();

Comment (UnknownEvil) [inline src/evolv/io/Brain.java]: Alright, I'll do that.

File: src/evolv/io/Brain.java [hunk 2]
```diff
@@ -209,4 +235,14 @@ private int neuronTextColor(double d) {
 			return this.evolvioColor.color(0, 0, 1);
 		}
 	}
+	
+	@SuppressWarnings("unused")
+	// Used based on NUM_EYES value
+	private final static int calculateBrainHeight() {
+		 if (Configuration.NUM_EYES * 3 + Configuration.MEMORY_COUNT + 3 < 9) {
```
Comment (NateUniAccount) [inline src/evolv/io/Brain.java]: Could this be calculated once and then the if check done, just makes it a bit easier to understand that it's the same value

**Comment by UnknownEvil** [comment]
Will be re-doing.

**Pull request closed without merging**